src/lsusb.rs

```diff
@@ -985,7 +985,7 @@ fn dump_ccid_desc(ccid: &CcidDescriptor, indent: usize) {
     );
     dump_value(
         ccid.max_clock,
-        "dwMaxiumumClock",
+        "dwMaximumClock",
```

Comment:
I'm on the fence to leave this one. Unlike the others, I think this was a conscious typo because it's in the lsusb source (or like minds!): https://github.com/gregkh/usbutils/blob/master/lsusb.c#L2185C18-L2185C33 . I'll add a comment if I keep it.

Reply:
umum!